app/src/main/res/xml/about.xml

```diff
@@ -18,8 +18,8 @@
             name="@string/owner_name"
             avatar="https://avatars2.githubusercontent.com/u/1151321"
             bio="@string/owner_bio"
-            email="@string/owner_email"
-            login="@string/owner_login"
+            email="davide@depau.eu"
+            login="Depau"
```

Comment:
`Attribouter ` doesn't parse strings as links from the resources. Also `email` attribute doesn't work correctly